web/src/components/AttributeRow/AttributeRow.tsx

```diff
@@ -34,11 +31,11 @@ const AttributeRow = ({
   onCreateAssertion,
   setIsCopied,
   index,
+  currentStep,
```

Comment:
Let's be more explicit about these new props. Like: what is open? or current step for what?

Reply:
done.
called it guidedTourStep andisGuidedTourOpen